tests/test_ui_domain_view.py

```diff
@@ -9,50 +9,26 @@
 
 from wtfis.clients.abuseipdb import AbuseIpDbClient
 from wtfis.clients.greynoise import GreynoiseClient
+
+# from wtfis.clients.ip2whois import Ip2WhoisClient  # TODO: Write tests for this
 from wtfis.clients.ipwhois import IpWhoisClient
 from wtfis.clients.shodan import ShodanClient
-from wtfis.clients.urlhaus import UrlHausClient
+
+# from wtfis.clients.urlhaus import UrlHausClient  # TODO: Write tests for this
 from wtfis.models.abuseipdb import AbuseIpDbMap
 from wtfis.models.greynoise import GreynoiseIpMap
 from wtfis.models.ip2whois import Whois as Ip2Whois
 from wtfis.models.ipwhois import IpWhoisMap
-from wtfis.models.passivetotal import Whois as PTWhois
 from wtfis.models.shodan import ShodanIpMap
-from wtfis.models.urlhaus import UrlHausMap
+
+# from wtfis.models.urlhaus import UrlHausMap  # TODO: Write tests for this
 from wtfis.models.virustotal import Domain, Resolutions
 from wtfis.models.virustotal import Whois as VTWhois
 from wtfis.ui.view import DomainView
 
 
 @pytest.fixture()
-def view01(test_data, mock_ipwhois_get):
-    """gist.github.com with PT whois. Complete test of all panels. Also test print()."""
-    resolutions = Resolutions.model_validate(
-        json.loads(test_data("vt_resolutions_gist.json"))
-    )
-
-    geoasn_pool = json.loads(test_data("ipwhois_gist.json"))
-    geoasn_client = IpWhoisClient()
-    geoasn_client._get_ipwhois = MagicMock(
-        side_effect=lambda ip: mock_ipwhois_get(ip, geoasn_pool)
-    )
-    geoasn_enrich = geoasn_client.enrich_ips(*resolutions.ip_list(3))
-
-    return DomainView(
-        console=Console(),
-        entity=Domain.model_validate(json.loads(test_data("vt_domain_gist.json"))),
-        resolutions=resolutions,
-        geoasn=geoasn_enrich,
-        whois=PTWhois.model_validate(json.loads(test_data("pt_whois_gist.json"))),
-        shodan=ShodanIpMap.model_validate({}),
-        greynoise=GreynoiseIpMap.model_validate({}),
-        abuseipdb=AbuseIpDbMap.model_validate({}),
-        urlhaus=UrlHausMap.model_validate({}),
-    )
```

Comment:
I'm afraid we still need this because it's used by the most complete / default test in the set. I would just use VT for whois, like [this](https://github.com/pirxthepilot/wtfis/blob/9bcc55bceb513d262c04edc104ee3d1278b19e72/tests/test_ui_domain_view.py#L67):

```
whois=VTWhois.model_validate(json.loads(test_data("vt_whois_gist.json"))),
```